circuits/lib/utils/claimUtils.circom

```diff
@@ -15,11 +15,11 @@ template getClaimSubjectOtherIden() {
 
     // get header flags from claim.
     component header = getClaimHeader();
-    for (var i=0; i<8; i++) { header.claim[i] <== claim[i]; }
+    header.claim <== claim;
```

Comment:
I think we can call getClaimHeader:
```
var claimType claimFlags ...
(claimType, claimFlags) = getClaimHeader()(claim);
```
I found this in the documentation: https://docs.circom.io/circom-language/anonymous-components-and-tuples/#syntax-and-semantics-of-anonymous-components

Reply:
Not working for `claimFlags[32]`